CIP-XXX/CIP-XXX.md

```diff
@@ -0,0 +1,1437 @@
+---
+CIP: ???
```

Comment:
```suggestion
CIP: 122
```
Please also change directory name, and also the link in the first comment in this thread to point to that directory or its `README.md` file.